remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java

```diff
@@ -302,4 +323,19 @@ private static Tag createTagWithNodeRef(final String tagName)
             .tag(tagName)
             .create();
     }
+
+    @Test
+    public void testAddTagsToNode()
```

Comment:
Is there value in a test for applying multiple tags? How about a test where the tag is not in the response from `findTaggedNodesAndCountByTagName`?

Reply:
Added testcase for multiple tags and findTaggedNodesAndCountByTagName returning empty list